auth-server/src/resolvers/mutation/resetPassword.ts

```diff
@@ -0,0 +1,15 @@
+import AuthService from '../../services/auth'
+import { UserInputError } from 'apollo-server'
+
+export default async (_, args) => {
+	const { token, newPassword } = args
+
```

Comment:
same as for the requestResetPassword, I'd check the jwt and get the user id from there so that you query your table on an index.

Reply:
Cannot use jwt. User is not logged in

Reply:
why not? I think a user that request a new email should be logged in. All you need is a username and password.